cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -97,6 +97,18 @@ def generate_video_file(filename, width=1280, height=720, duration=1, fps=25, co
     return [(width, height)] * total_frames, f
 
 
+def compare_datasets(test, expected, actual):
+    actual_copy = copy.deepcopy(actual)
```

Comment:
Why do we need to make a copy here?

Reply:
because dm_compare_datasets modifies a dataset. added a comment about it